projects/components/src/table/table.component.ts

```diff
@@ -93,9 +95,10 @@ import { CdkDragDrop, moveItemInArray } from '@angular/cdk/drag-drop';
             <cdk-header-cell
               [attr.data-column-index]="index"
               *cdkHeaderCellDef
-              [style.flex-basis]="columnDef.width"
-              [style.max-width]="columnDef.width"
-              [style.min-width]="columnDef.minWidth ?? columnDef.width ?? this.minColumnWidth"
+              [style.flex-basis]="this.getFlexBasis | htMemoize: columnDef.width"
```

Comment:
Why not have single method which return a json which can be passed as input to `ngStyle`? We'll get rid of these multiple memoize directives.

Reply:
+1